packages/map-template/README.md

```diff
@@ -110,3 +113,41 @@ Note that when using the React component, the properties should conform to JSX p
 |`external-IDs`|`etxternalIDs`|`array`|Array of external IDs which filters the map and shows a list of locations. Because of the way browsers work, you can not use External IDs with the `,`, `&`, `#` and `+`, character in them, as they are interpreted by the browser in a particular way. |
 |`tile-style`|`tileStyle`|`string`|Name of Tile Style to display on the map. |
 |`start-zoom-level`|`startZoomLevel`|`number`|The initial zoom level of the map. |
+|`supports-URL-parameters`|`supportsURLParameters`|`bool`|Indicates if the Map Template supports URL parameters. |
+
+## Using Query Parameters
+
+The Map Template supports using query parameters for all the properties provided by the MapsIndoorsMap component.
```

Comment:
Is this about the app? We should probably not mention that in readme for NPM since the app is not exported.

Reply:
Should the whole section be moved to the general README file for the Map template?

Reply:
Maybe it should be in both places.

Reply:
Updated the docs to fit correctly on each page.

Reply:
Also added npm links on the front page for the Map Template project and for the Midt as well